api/WEBGL_debug_renderer_info.json

```diff
@@ -15,10 +15,12 @@
             "version_added": "12"
           },
           "firefox": {
-            "version_added": "53"
+            "version_added": "53",
+            "notes": "Deprecated, and may be removed in a future release (see <a href='https://bugzil.la/1722782'>bug 1722782</a>)."
```

Comment:
Note, did not provide version (92) and normally we don't do so. I did provide bug because this is not a spec issue and I figure people need a place to query this.